android/app/src/main/res/drawable/profile_generic.xml

```diff
@@ -0,0 +1,7 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android" android:height="105.34979dp" android:viewportHeight="1280" android:viewportWidth="1215" android:width="100dp">
```

Comment:
How a user Material icon looks compared to this one? https://fonts.google.com/icons?selected=Material%20Symbols%20Outlined%3Aperson%3AFILL%401%3Bwght%40400%3BGRAD%400%3Bopsz%4024